pkg/endpoint/policy.go

```diff
@@ -293,21 +362,32 @@ func (e *Endpoint) regenerateConsumable(owner Owner, labelsMap *LabelsMap, repo
 		}
 	}
 
+	consumersToRm := map[policy.NIPortProto]bool{}
```

Comment:
consumers is a very weird name to use here

Reply:
`l4RulesToRm`?

Reply:
Maybe `rulesToDelete`